pkg/errors/sentry.go

```diff
@@ -0,0 +1,56 @@
+package errors
+
+import (
+	"github.com/cloudquery/cloudquery/pkg/core"
+	"github.com/cloudquery/cloudquery/pkg/plugin"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/getsentry/sentry-go"
+	"github.com/spf13/viper"
+)
+
+func CaptureError(err error, tags map[string]string) {
```

Comment:
Add `CaptureCommand(err error, commandName string)` to call this one with the `tags` auto filled in with the given command?